ratarmount.py

```diff
@@ -1166,6 +1174,11 @@ def _parseArgs(rawArgs: Optional[List[str]] = None):
         '-d', '--debug', type=int, default=1,
         help='Sets the debugging level. Higher means more output. Currently, 3 is the highest.')
 
+    advancedGroup.add_argument(
+        '--force-libarchive', action='store_true', default=False,
+        help='Use libarchive to handle .tar, .rar, an .zip files.')
```

Comment:
```suggestion
        help='Use libarchive to also handle TAR, RAR, and ZIP archives.')
```
I would say with leading dot refers to file extensions but ratarmount tries to be mostly extension agnostic, so call them by their names.